runsc/cgroup/cgroup.go

```diff
@@ -295,19 +345,46 @@ func NewFromPath(cgroupsPath string) (*Cgroup, error) {
 		}
 	}
 	own := make(map[string]bool)
+
 	return &Cgroup{
 		Name:    cgroupsPath,
 		Parents: parents,
 		Own:     own,
 	}, nil
 }
 
+func (c *Cgroup) cgroupv2Manager() (*cgroupV2Manager, error) {
+	if c.v2manager != nil {
```

Comment:
This might be racy if cgroupv2Manager() is called from two places simultaneously.

Two alternative ways to fix it
1. Use `sync.Once()` to init `v2manager`
2. Use mutex lock.

I'd use sync.Once.